js/services/scrobbler.js

```diff
@@ -6,13 +6,33 @@
 
       service.getSongInformation = function () {
         var deferred = $q.defer();
+        var songInformation = {};
         if (config.lastfm.user.length) {
```

Comment:
I think that rather than using 'length' property, it would be better to use it like this.
`if (config.lastfm && config.lastfm.user)`